### ISSUE TITLE
Svelte 5 migration

### ISSUE THREAD

**Issue by vlrevolution** [framework7io/framework7#4296]
I guess we should start discussing migration to Svelte 5 here. Anybody working on it yet?

**Comment by Muhammed-Rahif** [comment]
Can the svelte 5 migration script handle this? https://svelte.dev/docs/svelte/v5-migration-guide#Migration-script

**Comment by vlrevolution** [comment]
We should also keep backward compatibility with Svelte 4 because not everybody will transition right away due to some core libraries not yet migrated

**Comment by CodeDoes** [comment]
We just need to change the mount, for new projects (which automatically use svelte5).
```js
import {mount} from "svelte"
const app = mount(App,{
  target: document.getElementById('app'),
});
```

**Comment by raiyanihiren** [comment]
@CodeDoes Right, but it's still not refreshing the DOM in Svelte when the variable changes. This was working in Svelte 4.

**Comment by michielwittkampf** [comment]
Progress on a version for Svelte 5 will be much appreciated 👍

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
